internal/handler/command_test.go

```diff
@@ -452,3 +466,9 @@ func TestCommandHandler(t *testing.T) {
 		})
 	}
 }
+
+func newRequestWithContentType(contentType string) *http.Request {
```

Comment:
I do not see any tests that cover using the `application/cbor` content type nor an empty, or invalid content type. Can we add some tests to cover those cases?